lisa/operating_system.py

```diff
@@ -762,6 +762,8 @@ def add_repository(
             expected_exit_code=0,
             expected_exit_code_failure_message="fail to add repository",
         )
+        if not isinstance(self._node.os, Ubuntu):
```

Comment:
Is it to fix for debian only? If so,
1. Add comments to explain it.
2. Check it like `if type(self._node.os) == Debian)`.

Reply:
updated